lib/cf/bridge/src/index.js

```diff
@@ -137,6 +137,25 @@ const statistics = {
   }
 };
 
+const errors = {
+  missingToken: false,
+  noReportEverHappened: true,
+  consecutiveFailures: 0,
+  lastError: '',
+  lastErrorTimestamp: ''
+};
+
+const registerError = (message, error, name, time) => {
+  errors.lastError = message;
+  errors.lastErrorTimestamp = new Date().toISOString();
+
+  const msg = util.format('messase: %j error: %j', message, error);
```

Comment:
I believe we can use the format functionality of the debug module directly and use:
```
edebug('%s error: %j');
```